app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -60,37 +61,22 @@ class BrowserWebViewClient(
     private val appCoroutineScope: CoroutineScope,
     private val dispatcherProvider: DispatcherProvider,
     private val emailInjector: EmailInjector
-) : WebViewClient() {
+) : WebViewClientCompat() {
 
     var webViewClientListener: WebViewClientListener? = null
     private var lastPageStarted: String? = null
 
-    /**
-     * This is the new method of url overriding available from API 24 onwards
-     */
-    @RequiresApi(Build.VERSION_CODES.N)
     override fun shouldOverrideUrlLoading(view: WebView, request: WebResourceRequest): Boolean {
         val url = request.url
-        return shouldOverride(view, url, request.isForMainFrame)
-    }
-
-    /**
-     * * This is the old, deprecated method of url overriding available until API 23
-     */
-    @Suppress("OverridingDeprecatedMember")
-    override fun shouldOverrideUrlLoading(view: WebView, urlString: String): Boolean {
```

Comment:
we don't need this anymore, the new WebViewClientCompat provides backwards compatibility for `shouldOverrideUrlLoading(view: WebView, request: WebResourceRequest)` up to Android L.